app/services/gemini_analyzer.py

```
@@ -0,0 +1,105 @@
+    note: str | None = None
+
+
+@dataclass(frozen=True)
+class AnalyzedRecipe:
+    ingredients: list[IngredientData]
+    steps: list[str]
+    total_cost: int
+    servings: int
+
+
+def analyze_recipe_from_audio(auido_path: str) -> AnalyzedRecipe:
```

Ruff (0.14.14)

[warning] Unused function argument: `auido_path`

(ARG001)